pytorch_lightning/loops/dataloader/evaluation_loop.py

```diff
@@ -272,16 +272,67 @@ def _on_evaluation_epoch_end(self) -> None:
         self.trainer.logger_connector.on_epoch_end()
 
     def _print_results(self, results: List[_OUT_DICT], stage: RunningStage) -> None:
-        # TODO: this could be updated to look nicer
-        from pprint import pprint
-
-        print("-" * 80)
-        for i, metrics_dict in enumerate(results):
-            print(f"DATALOADER:{i} {stage.upper()} RESULTS")
-            pprint(
-                {
-                    k: (v.item() if v.numel() == 1 else v.tolist()) if isinstance(v, torch.Tensor) else v
-                    for k, v in metrics_dict.items()
-                }
-            )
-            print("-" * 80)
+        if _RICH_AVAILABLE:
+            from rich.console import Console
```

Comment:
Let's move those methods into utlity.

Reply:
What do you mean exactly with this?

Reply:
He means that after we are set on the design, we should refactor as much unto utility methods we use to test, and move them out of this file.